### PR TITLE
Remove "open source" langauge throughout

### LINKED ISSUE #40: n8n is not open source and your project is gaslighting its users
This behavior is not acceptable. Even the Commons Clause itself tells you not to describe your software as open source, see the FAQ: https://commonsclause.com/

The Apache license is a well known and respected license, and the "Commons Clause" leverages the language of the also well known "Creative Commons", together creating a false impression that this software is distributed by a respectable group of developers. You cannot rely on putting its name there to correct your lies.

Your software is rightly called "source available". Do not gaslight us. In the words of [Randall Monroe](https://xkcd.com/971/):

> I just noticed CVS has started stocking homeopathic pills on the same shelves with--and labeled similarly to--their actual medicine. Telling someone who trusts you that you're giving them medicine, when you know you're not, because you want their money, isn't just lying--it's like an example you'd make up if you had to illustrate for a child why lying is wrong.

Note: If you lock this issue you are a liar *and* a coward. Hear the anger of those you've wronged.

Edit: [this link](https://docs.n8n.io/#/faq?id=is-n8n-really-open-source) is useful for onlookers wondering I'm angry out of the gate.

### PR DESCRIPTION
**Pull request by ddevault**
This project is not open source. I could not find the source for the
website, but it will have to be updated as well.

Fixes #40

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ![n8n.io - Workflow Automation](https://raw.githubusercontent.com/n8n-io/n8n/master/docs/images/n8n-logo.png)
 
-n8n is a free node based "Open Source" (with Commons Clause)
+n8n is a free node based source-availalbe
```
Comment (PawelGlow) [inline README.md]: @ddevault typo: should be "source-available"

**Comment by kfogel** [comment]
The source for the web site is at https://github.com/n8n-io/n8n-website-github, by the way.

**Comment by janober** [comment]
@kfogel FYI Actually that is not the source of the website. That is the source of the website I had live a few months ago and simply forgot to remove from Github. Only realised when you opened the issue there which kind of made it now impossible for me to remove it like planned. The source of the current website is not on Github.

File: README.md (hunk 1 above)
Comment (ddevault) [reply README.md]: Thanks, fixed.

**Comment by kfogel** [comment]
Hey, @janober.  I didn't quite understand the part about "which kind of made it now impossible for me to remove it like planned"?

**Comment by janober** [comment]
@kfogel I had planned to remove the old website repository but it would feel wrong now because there are also comments about this not yet resolved "open source" issue. So to not remove anything related to that I keep the repository up there for now.

**Comment by leonardlin** [comment]
interesting, I did not know that someone owns "open source" and is the global ruler over what it may mean.

**Comment by Frederik-Baetens** [comment]
I think the ui can be found in the main repo here: https://github.com/n8n-io/n8n/tree/master/packages/editor-ui

Is that not the case?

To all the nice people working on this project, thank you for making such a great open source project.

**Comment by janober** [comment]
Close this issue as wording got changed and the related issue #40 got also closed.
Uses now https://faircode.io

**Pull request closed without merging**